go-chaos/cmd/root.go

```diff
@@ -39,18 +39,25 @@ var (
 )
 
 var Verbose bool
+var KubeConfigPath string
+var Namespace string
+
 var rootCmd = &cobra.Command{
 	Use:   "zbchaos",
 	Short: "Zeebe chaos is a chaos experiment tool for Zeebe",
 	Long: `A chaos experimenting toolkit for Zeebe.
     Perfect to inject some chaos into your brokers and gateways.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		internal.Verbosity = Verbose
+		internal.Namespace = Namespace
+		internal.KubeConfigPath = KubeConfigPath
```

Comment:
I guess we need also some defaults ?

Reply:
Default for the config path is set here: https://github.com/zeebe-io/zeebe-chaos/pull/193/files#diff-464316c02acb79ad2acc14ad37881cf54b046e8f496271a18a68502f674944b0R79-R88